logstash-core/lib/logstash/runner.rb

```diff
@@ -286,6 +286,10 @@ def execute
     # override log level that may have been introduced from a custom log4j config file
     LogStash::Logging::Logger::configure_logging(setting("log.level"))
 
+    if checkLogConfigurationContainsJavascriptUsage
```

Comment:
nit: pbly too much Java<->Ruby the rest of the file follows `ruby_naming_conventions` :wink: